### DISCUSSION TITLE
Bug on collapsed headers on full width

### DISCUSSION THREAD

**Discussion by napest** [outline/outline#3557] [General] [answered]
Hi!

If I select Full width option collapsed headers no longer expand. I'm using Chrome 100.0.4896.127 on macos and 0.63.0 version, but I see the same behavior in cloud version as well. Is it a bug?

https://user-images.githubusercontent.com/18447985/169378453-616b6267-a27e-4ca9-ae94-70c4dc22fe30.mov

**Comment by almereyda** [comment]
Confirmed on GNOME Web 41.4 (WebKitGTK 2.36.1).

**Reply by almereyda** [reply]
Also confirmed on Firefox 100.0 ... ah, I see. It's closed in 90ca8655af84fbc62771c1548b751121ce3e87f0

**Comment by tommoor** [comment, marked as answer]
Closed in https://github.com/outline/outline/issues/3558
